### PR TITLE
Page improvement: 404 page.

### PR DESCRIPTION
**Pull request by andreyzb**
Patch adds block visibility options for 404/403 status codes

Original Issue, this PR is going to fix: https://y-usa.visualstudio.com/Open%20Y%20Product%20Development/_workitems/edit/32035/
https://fivejars.atlassian.net/browse/YUOPENY-16

## Steps for review
- [ ] Login as Admin
- [ ] Go to Structure > Block layout (`/admin/structure/block`) and edit any existing block 
- [ ] Verify that Visibility > Error pages vertical tab exists and saves correctly

## General checks
- [x] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] Try to use Conventional commit messages accordingly to the standard https://www.conventionalcommits.org/en/v1.0.0/#specification
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/9.x-2.x/docs) has been updated according to PR changes.
- [x] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Upgrade%20path.md).
- [x] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [x] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Comment by andreyzb** [comment]
@podarok Please review

File: composer.json [hunk 1]
```diff
@@ -211,7 +211,8 @@
             "drupal/core": {
                 "1236098 - Notice: Undefined index in _color_rewrite_stylesheet()": "https://www.drupal.org/files/issues/undefined-index-in-_color_rewrite_stylesheet-1236098-37.patch",
                 "2484693 - Telephone Link field formatter InvalidArgumentException with 5 digits or fewer in the number": "https://www.drupal.org/files/issues/2484693-54.patch",
-                "2862702 - PrepareModulesEntityUninstallForm::formTitle does not exist": "https://www.drupal.org/files/issues/2862702-3.patch"
+                "2862702 - PrepareModulesEntityUninstallForm::formTitle does not exist": "https://www.drupal.org/files/issues/2862702-3.patch",
+                "2245767 - Allow blocks to be configured to show/hide on 403/404 pages": "https://www.drupal.org/files/issues/2021-11-29/2245767-93.patch"
```
Comment (podarok) [inline composer.json:215]: Let's have a comment with RTBC status in https://www.drupal.org/node/2245767

Comment (podarok) [reply composer.json:215]: @andreyzb

Comment (andreyzb) [reply composer.json:215]: @podarok https://www.drupal.org/project/drupal/issues/2245767#comment-14742053

**Pull request merged**